include/aos/common/config.hpp

```diff
@@ -533,6 +533,13 @@
 #define AOS_CONFIG_MONITORING_AVERAGE_WINDOW_SEC 60
 #endif
 
+/**
+ * Monitoring alert processor system resources size.
+ */
+#ifndef AOS_CONFIG_MONITORING_ALERT_PROCESSOR_SYSTEM_RESOURCES_SIZE
```

Comment:
You don't need config const for it. Number of resources are well defined value. Just make private constant.